### ISSUE TITLE
EventEmitter.removeListener is deprecated

### ISSUE THREAD

**Issue by stanwolverine** [100mslive/react-native-hms#863]
Fix Warning ```EventEmitter.removeListener: Method has been deprecated. Please instead use `remove()` on the subscription returned by `EventEmitter.addListener` ```

**Comment by akanz** [comment]
I'm having this issue. even when the call has been disconnected the listeners are still there

**Comment by ygit** [comment]
@akanz Which version of React Native are you using?
100ms supports RN version 0.63.x & above.
The messages you are getting are Warning logs due to the usage of removeListener API which was deprecated in RN version 0.65.0.

**Comment by swaaj** [comment]
I am getting an Error (and not a warning) : 
````
Leave or Destroy Error:  [TypeError: HmsEventEmitter.removeListener is not a function. (In 'HmsEventEmitter.removeListener(_HMSUpdateListenerActions.HMSUpdateListenerActions.ON_JOIN, _this.onJoinListener)', 'HmsEventEmitter.removeListener' is undefined)]
````

I am using React Native 0.70.x

**Comment by stanwolverine** [comment]
Hi @swaaj, This issue will be fixed in upcoming release this week.

**Comment by ygit** [comment]
hey @swaaj @akanz 
We have released version [1.2.0](https://www.npmjs.com/package/@100mslive/react-native-hms) which fixes this issue. Do update & let us know.
